addons/bus/__manifest__.py

```diff
@@ -13,6 +13,7 @@
         'web.assets_common': [
             'bus/static/src/*.js',
             'bus/static/src/services/**/*.js',
+            ('remove', 'bus/static/src/services/assets_watchdog_service.js'),
```

Comment:
Note that this might not be needed anymore as assets_common are not part of the asset frontend anymore.
That change was merged after this PR was created.

But let's see if @tsm-odoo sees something wrong about this PR first.